docs/generating-a-provider.md

```diff
@@ -167,14 +161,15 @@ be quite similar for any other Terraform provider.
        "github.com/upbound/upjet/pkg/types/conversion/cli"
        "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 
-   +   "github.com/crossplane-contrib/provider-jet-github/config/branch"
-   +   "github.com/crossplane-contrib/provider-jet-github/config/repository"
+   +   "github.com/myorg/provider-github/config/branch"
+   +   "github.com/myorg/provider-github/config/repository"
     )
 
     func GetProvider() *tjconfig.Provider {
        ...
        for _, configure := range []func(provider *tjconfig.Provider){
                // add custom config functions
+   -           null.Configure,
```

Comment:
nit: Why do we have the literal `null` here?

Reply:
It's because the provider name is `null`, i.e. `hashicorp/null`.